app/ui/components/bannerHomePage.tsx

```diff
@@ -0,0 +1,58 @@
+import { Box, Flex, Button, ButtonGroup } from '@chakra-ui/react';
+
+export default function BannerHomePage() {
+  return (
+    <Flex bg="tomato" w="100%">
+      <Box bg="#2945FF" w="50%" color="white">
+        <Box my={75} ml={125}>
+          <h2
+            style={{
+              fontFamily: 'Inter',
+              fontWeight: 500,
+              fontSize: '4rem',
+              margin: '2rem 0',
+            }}
```

Comment:
Check if there is not already a component for title in chakra ui.

Besides don't use style attributes but styled component instead.

This is the same everytime you use style attributes, it is Bad !